code/_onclick/hud/generic_dextrous.dm

```diff
@@ -63,13 +63,13 @@
 		return
 	var/mob/living/D = mymob
 	if(hud_version != HUD_STYLE_NOHUD)
-		for(var/obj/item/I in D.held_items)
-			I.screen_loc = ui_hand_position(D.get_held_index_of_item(I))
-			D.client.screen += I
+		for(var/obj/item/item in D.held_items)
+			item.screen_loc = ui_hand_position(D.get_held_index_of_item(item), item)
```

Comment:
?

Reply:
You can take a look at the doc comment of the proc if you have no idea why there's a second arg now.

Reply:
Feels like this could be handled inside the proc but thats a nitpick at best